src/common/component.rs

```diff
@@ -170,6 +170,10 @@ impl Components {
     /// let mut components = Components::new();
     /// components.refresh_list();
     /// ```
+    ///
+    /// ⚠️ This function is currently useless on every call after the first one on
```

Comment:
I would instead make this function do nothing on macOS at all until the bug is fixed.

Reply:
I think the sentence is just misleading. The crash has been resolved, the comment just doesn't yet state that this is the case.